requirements.txt

```
@@ -3,3 +3,5 @@ pandas
 numpy
 scikit_learn >= 1.0.0
 julia >= 0.6.0
+click>=8.0.4
```

```suggestion
click>=7.0.0
```

Looks like the old version has the same API so we could lower this (in case another package in the same env requires an old version).